views/shop/home.ejs

```diff
@@ -1,10 +1,10 @@
-<head> 
```

Comment:
i'll approve these changes because we are time sensitive but next pull request you have to fix this ugly indentation after i fixed it